extensions/metrics/README.md

```diff
@@ -80,86 +88,85 @@ information for the Node.js process as well as LoopBack method invocations.
 
 <details>
 <summary>Example of plain text data</summary>
-<pre><code>
+<pre>
 # HELP process_cpu_user_seconds_total Total user CPU time spent in seconds.
 # TYPE process_cpu_user_seconds_total counter
 process_cpu_user_seconds_total 0.132181 1564508354524
-</code></pre>
-
-<pre><code>
+</pre>
+<pre>
```

Comment:
This is still introducing extra empty space between code blocks.

<img width="911" alt="Screen Shot 2019-09-19 at 16 40 00" src="https://user-images.githubusercontent.com/1140553/65254197-31956980-dafc-11e9-827a-d094a71bfdff.png">

Is it your intention to visually delimit these blocks?